### PR TITLE
Misc: Use the ^ operator instead of math.pow

### PR DESCRIPTION
**Pull request by NiLuJe**
As discussed in #9544 ;).

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9550)

### CODE REVIEW

File: frontend/ui/widget/imagewidget.lua [hunk 1]
```diff
@@ -33,7 +33,7 @@ local logger = require("logger")
 local function get_dpi_scale()
     local size_scale = math.min(Screen:getWidth(), Screen:getHeight()) / 600
     local dpi_scale = Screen:scaleByDPI(1)
-    return math.pow(2, math.max(0, math.log((size_scale+dpi_scale)/2)/0.69))
+    return math.max(0, math.log((size_scale+dpi_scale)/2)/0.69)^2
```
Comment (zwim) [inline frontend/ui/widget/imagewidget.lua:36]: Faste would be
```
return math.max(0,  (math.log((size_scale+dpi_scale)/2)/0.69)^2 )
```

File: frontend/device/sysfs_light.lua [hunk 2]
```diff
@@ -91,14 +91,14 @@ function SysfsLight:setNaturalBrightness(brightness, warmth)
...
         end
         if warmth > 0 then
-            red = math.min(self.red_gain * math.pow(brightness, self.exponent) *
-                           math.pow(warmth, self.exponent) + self.red_offset, 255)
-            green = math.min(self.green_gain * math.pow(brightness, self.exponent) *
-                             math.pow(warmth, self.exponent) + self.green_offset, 255)
+            red = math.min(self.red_gain * brightness^self.exponent *
+                           warmth^self.exponent + self.red_offset, 255)
+            green = math.min(self.green_gain * brightness^self.exponent *
+                             warmth^self.exponent + self.green_offset, 255)
```
Comment (zwim) [inline frontend/device/sysfs_light.lua:101]: Faster would be
```
local brightness_warmth_exp = (brightness * warmth)^self.exponent
red = math.min(self.red_gain * brightness_warmth_exp + self.red_offset, 255)
green = math.min(self.green_gain * brightness_warmth_exp + self.green_offset, 255)
```

**Review by zwim — COMMENTED** [review]
Some eyecatchers, not tested.

File: frontend/device/sysfs_light.lua [hunk 3]
```diff
@@ -91,14 +91,14 @@ function SysfsLight:setNaturalBrightness(brightness, warmth)
         if brightness > 0 then
             -- On Nickel, the values for white/red/green are roughly linearly dependent
             -- on the 4th root of brightness and warmth.
-            white = math.min(self.white_gain * math.pow(brightness, self.exponent) *
-                             math.pow(100 - warmth, self.exponent) + self.white_offset, 255)
+            white = math.min(self.white_gain * brightness^self.exponent *
+                             (100 - warmth)^self.exponent + self.white_offset, 255)
```
Comment (zwim) [inline frontend/device/sysfs_light.lua:95]: Save one exponentation
```
white = math.min(self.white_gain * (brightness * (100 - warmth))^self.exponent + self.white_offset, 255)
```

File: frontend/device/sysfs_light.lua (hunk 2 above)
Comment (NiLuJe) [reply frontend/device/sysfs_light.lua:101]: FWIW, I'm assuming the compiler takes care of this, but it's harmless and won't affect readability, so, why not ;).

**Review by NiLuJe — COMMENTED** [review]
> Reviewed 1 of 4 files at r1, 1 of 1 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/koreader/koreader/9550)* status: all files reviewed, 3 unresolved discussions (waiting on @zwim)

___
*[`frontend/device/sysfs_light.lua` line 95 at r1](https://reviewable.io/reviews/koreader/koreader/9550#-NCbDPIJmJWZnDd1Nmrw-r1-95:-NChEta9CMKN7KgYHjxe:b-896fix) ([raw file](https://github.com/koreader/koreader/blob/62e2b4cb28de983c32d28e34384990484daaed32/frontend/device/sysfs_light.lua#L95)):*
<details><summary><i>Previously, zwim wrote…</i></summary><blockquote>

Save one exponentation
```
white = math.min(self.white_gain * (brightness * (100 - warmth))^self.exponent + self.white_offset, 255)
```
</blockquote></details>

Done.
___
*[`frontend/ui/widget/imagewidget.lua` line 36 at r1](https://reviewable.io/reviews/koreader/koreader/9550#-NCbDPIJmJWZnDd1Nmrx-r1-36:-NChEuXE0OevZW1iFSfA:b-896fix) ([raw file](https://github.com/koreader/koreader/blob/62e2b4cb28de983c32d28e34384990484daaed32/frontend/ui/widget/imagewidget.lua#L36)):*
<details><summary><i>Previously, zwim wrote…</i></summary><blockquote>

Faste would be
```
return math.max(0,  (math.log((size_scale+dpi_scale)/2)/0.69)^2 )
```

</blockquote></details>

Done.

**Pull request merged**